content/07.prior-knowledge.md

```diff
@@ -1,14 +1,23 @@
 ### Build upon prior knowledge and indirectly related data {.page_break_before}
 
 Rare diseases often lack large, normalized datasets, limiting our ability to study key attributes of these diseases. 
-One strategy to overcome this is to integrate and explore rare disease information alongside other knowledge by combining a variety of different data types. By using several data modalities, it may be possible to gain a better understanding of rare diseases (e.g., identifying novel genotype-phenotype relationships or opportunities for drug repurposing).
+One strategy to overcome this is to integrate and explore rare disease information alongside other knowledge by combining a variety of different data types. By using several data modalities (such as curated pathway, genetic data, or other data types) , it may be possible to gain a better understanding of rare diseases (e.g., identifying novel genotype-phenotype relationships or opportunities for drug repurposing).
```

Comment:
```suggestion
One strategy to overcome this is to integrate and explore rare disease information alongside other knowledge by combining a variety of different data types. 
By using several data modalities (such as curated pathway, genetic data, or other data types), it may be possible to gain a better understanding of rare diseases (e.g., identifying novel genotype-phenotype relationships or opportunities for drug repurposing).
```